flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/delegation/BatchPlanner.scala

```diff
@@ -85,10 +86,14 @@ class BatchPlanner(
     }
     val optimizedRelNodes = optimize(sinkRelNodes)
     val execNodes = translateToExecNodePlan(optimizedRelNodes)
-    val transformations = translateToPlan(execNodes)
-    val batchExecutor = new BatchExecutor(getExecEnv)
-    batchExecutor.setTableConfig(getTableConfig)
-    val streamGraph = batchExecutor.generateStreamGraph(transformations, "")
+
+    val plannerForExplain = createPlannerForExplain()
+    val transformations = translateToPlan(execNodes, plannerForExplain)
```

Comment:
I think its unnecessary to change the signature of `translateToPlan` method.

Isn't it enought to call `plannerForExplain.translateToPlan(execNodes)`?